openforcefield/utils/toolkits.py

```diff
@@ -465,20 +465,12 @@ def to_file_obj(self, molecule, file_obj, file_format):
             The format for writing the molecule data
 
         """
-        from openeye import oechem
         from openforcefield.utils import temporary_directory, temporary_cd
-
-        oemol = self.to_openeye(molecule)
-
         # TODO: This is inefficiently implemented. Is there any way to attach a file-like object to an oemolstream?
         with temporary_directory() as tmpdir:
             with temporary_cd(tmpdir):
                 outfile = 'temp_molecule.' + file_format
-                ofs = oechem.oemolostream(outfile)
-                openeye_format = getattr(oechem, 'OEFormat_' + file_format)
-                ofs.SetFormat(openeye_format)
-                oechem.OEWriteMolecule(ofs, oemol)
-                ofs.close()
+                self.to_file(molecule, outfile, file_format)
```

Comment:
Nice, I like that we are using one function for both files and file objects now.